backend/capellacollab/core/database/migration.py

```diff
@@ -86,16 +87,21 @@ def create_tools(db):
     tools.create_tool(db, capella)
     tools.create_tool(db, papyrus)
 
-    tools.create_version(db, capella.id, "6.0.0", True)
+    default_version = tools.create_version(db, capella.id, "6.0.0", True)
     tools.create_version(db, capella.id, "5.2.0")
     tools.create_version(db, capella.id, "5.0.0")
 
     tools.create_version(db, papyrus.id, "6.1")
     tools.create_version(db, papyrus.id, "6.0")
 
-    tools.create_nature(db, capella.id, "model")
+    default_nature = tools.create_nature(db, capella.id, "model")
     tools.create_nature(db, capella.id, "library")
 
     tools.create_nature(db, papyrus.id, "UML 2.5")
     tools.create_nature(db, papyrus.id, "SysML 1.4")
     tools.create_nature(db, papyrus.id, "SysML 1.1")
+
+    for model in models.get_all_models(db):
+        models.set_tool_details_for_model(
```

Comment:
This doesn't set the tool (only the version and nature). Still get the same issue with the test database: 
```
pydantic.error_wrappers.ValidationError: 3 validation errors for ResponseModel
response -> 0 -> tool
```

Reply:
Fixed normally